druid-shell/src/platform/windows/window.rs

```diff
@@ -103,9 +102,8 @@ pub enum PresentStrategy {
     FlipRedirect,
 }
 
-#[derive(Default)]
 pub struct WindowHandle {
-    // Note: this clone of the dwrite factory might move into WinCtxImpl.
+    //FIXME: why is this optional?
```

Comment:
Looks like it's not needed. A long time ago, there was probably a two-stage initialization, where window state structures were created before window creation proper, then filled in after.

Note that when https://github.com/linebender/piet/pull/139 lands, we should be able to just derive `Clone` for `WindowHandle` here.